_docs/05.1-faq.markdown

```diff
@@ -4,4 +4,13 @@ permalink: docs/faq.html
 toplevel: "Resources"
 ---
 
-Coming Soon!
\ No newline at end of file
+You Ask, We Answer:
```

Comment:
i feel like we should leave it as `Coming Soon!` until we have solid content for these pages.